src/main/java/mk/edu/uklo/fikt/fiktexamweb/model/Test.java

```diff
@@ -0,0 +1,20 @@
+package mk.edu.uklo.fikt.fiktexamweb.model;
+
+import javax.persistence.*;
```

Comment:
Don't use imports with "*". If someone in the future adds some class/interface or any java type that has the same name in some of the packages that are imported the code will stop compiling. Always use full imports.